app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt

```diff
@@ -130,6 +151,14 @@ fun AddActivityScreen(tripsViewModel: TripsViewModel, navigationActions: Navigat
       return
     }
 
+    if (startTime == null && endTime == null)
+        Toast.makeText(context, "No times selected, defaults to 00:00 - 23:59", Toast.LENGTH_SHORT)
+            .show()
```

Comment:
This toast now gets displayed even when we create a draft (with both startTime and endTime 0,0)
Please make sure these toasts get displayed only when we have a date.

Also, a toast saying "No date assigned, created as draft" if we do not assign a date would be nice to have